client/task-list/tasks/payments/RemotePayments/index.js

```diff
@@ -45,10 +46,24 @@ export const RemotePayments = ( { query } ) => {
 
 		const enabled = new Map();
 		const additional = new Map();
+		let wcPay = null;
 		const recommendations = select( ONBOARDING_STORE_NAME )
 			.getPaymentMethodRecommendations()
 			.reduce( ( map, gateway ) => {
 				map.set( gateway.key, gateway );
+
+				// WCPay is handled separately when not installed and configured
+				if (
+					gateway.key === 'woocommerce_payments' &&
+					! (
+						paymentGateways[ gateway.key ] &&
+						paymentGateways[ gateway.key ].needs_setup !== true
```

Comment:
Maybe a little easier to visually parse:

```suggestion
						! paymentGateways[ gateway.key ].needs_setup
```